api2-war/src/main/java/eu/europeana/api2/v2/web/controller/ObjectController.java

```diff
@@ -138,7 +145,12 @@ public ModelAndView record(@PathVariable String collectionId,
     @SwaggerIgnore
     @GetMapping(value = {"/context.jsonld", "/context.json-ld"}, produces = MEDIA_TYPE_JSONLD_UTF8)
     public ModelAndView contextJSONLD(@RequestParam(value = "callback", required = false) String callback) {
-        String jsonld = JSONUtils.toString(jsonldContext);
+        String jsonld = null;
+        try {
+            jsonld = com.github.jsonldjava.utils.JsonUtils.toString(jsonldContext);
+        } catch (IOException e) {
+           LOG.error("Error in Json serialization", e);
+        }
```

Comment:
Why is this now wrapped in a try/catch? If there is an error we will now probably return an empty 200 response which is not what we want.

Reply:
removed